From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 07 Feb 2005 14:38:32 -08:00
From: matthew@wil.cx
Subject: [PATCH] PCI: Make pci_claim_resource __devinit

ia64 calls pci_claim_resource() from pcibios_fixup_bus(), which is
__devinit, so pci_claim_resource() needs to be __devinit too.

Signed-off-by: Matthew Wilcox <matthew@wil.cx>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/drivers/pci/setup-res.c b/drivers/pci/setup-res.c
--- a/drivers/pci/setup-res.c	2005-03-02 23:05:16 -08:00
+++ b/drivers/pci/setup-res.c	2005-03-02 23:05:16 -08:00
@@ -90,7 +90,7 @@
 		new & ~PCI_REGION_FLAG_MASK));
 }
 
-int __init
+int __devinit
 pci_claim_resource(struct pci_dev *dev, int resource)
 {
 	struct resource *res = &dev->resource[resource];
